internal/weaver/remoteweavelet.go

```diff
@@ -392,6 +391,21 @@ func (w *RemoteWeavelet) GetLoad(*protos.GetLoadRequest) (*protos.GetLoadReply,
 
 // UpdateComponents implements the conn.WeaverHandler interface.
 func (w *RemoteWeavelet) UpdateComponents(req *protos.UpdateComponentsRequest) (*protos.UpdateComponentsReply, error) {
+	var errs []error
```

Comment:
Do/should we retry this on the deployer side?

Reply:
Great question. I still haven't wrapped my head around exactly what should be retried. This case is tricky because UpdateComponents currently only returns an error if a deployer asks to start a component that wasn't registered. So, retrying would yield the same error. 

Creating the components themselves happens in a goroutine and can also fail, and right now we don't have a good way to report that error to the deployer. I'll have to think more about that.